### DISCUSSION TITLE
Why torch.softmax was not needed on logits for ComputerVision Model_0 ON FashionMNIST data?

### DISCUSSION THREAD

**Discussion by ankitgooner** [mrdbourke/pytorch-deep-learning#798] [Q&A] [answered]
While preparing the accuracy fn parameters on test data  argmax was directly used without applying softmax prior to it unlike multi class model training eariler
Can someone please assist with the explanation.?

**Comment by mrdbourke** [comment, marked as answer]
Hey @ankitgooner , 

In short, softmax is a transformation from logits -> prediction probabilities that range from 0 to 1.

Softmax makes the logits easier to understand from a human perspective.

However, it's not 100% necessary to use softmax when performing argmax on the logits to get the predictions.

See a fuller explanation here: https://github.com/mrdbourke/pytorch-deep-learning/discussions/314

**Reply by ankitgooner** [reply]
thanks for the link to fuller explanation as well @mrdbourke .
Apologies i could not locate that during the search.
